Sources/Modules/Yggdrasil/Everglow.Yggdrasil.csproj

```diff
@@ -5,4 +5,7 @@
   <ItemGroup>
 		<Solaestas-ResourceFile Include="YggdrasilTown\**\*.json" />
   </ItemGroup>
+  <ItemGroup>
+    <None Remove="Furnace\Buffs\MelterGearCooldown.png" />
```

Comment:
These are accidental change, which should be deleted

Reply:
fixed